src/devtools/plugins.js

```diff
@@ -8,3 +10,19 @@ Vue.use(VTooltip, {
   },
   defaultOffset: 2
 })
+
+const currentLocale = 'en'
+const locales = require.context('./locales')
+const keyboardReg = /\[(\w+)\]/g
+const iconReg = /\|(\w+)\|/g
+Vue.use(VI18n, {
+  strings: locales(`./${currentLocale}`).default,
+  defaultValues: {
+    keys
+  },
+  replacer: text => {
+    text = text.replace(keyboardReg, '<span class="keyboard">$1</span>')
```

Comment:
I don't like this solution. Now whenever `[]` or `||` appears somewhere in translation, it's going to wrap this part with one of those elements. As for the icon - if it's a generic app-wide solution I'm ok with it, but probably with better interpolation signs. e.g `<<iconName>>`, or at least `||iconName||` - though the first one more readable I guess.
But keyboard is rather case specific thing, and I wouldn't put it here. I'm thinking about alternative ways, and perhaps it would be better to make a dedicated file just for tooltips content, and import keys explicitly there.
That would leave us with translations in tact, without extra concerns.

Reply:
We are going to have a lot of keyboard shortcuts in the strings--more than icons 😛